README.md

```diff
@@ -20,13 +20,18 @@ Quick Links:
 ## How does it work?
 
 Cloud Run Release Manager periodically checks for new revisions in the services
-that opted-in for gradual rollouts. If a new revision with no traffic is found,
-the Release Manager automatically assigns it some initial traffic. This new
-revision is labeled `candidate` while the previous revision serving traffic is
-labeled `stable`.
+that opted-in for gradual rollouts.
 
-Depending on the candidate's health, traffic to the `candidate` is increased
-or traffic to the candidate is dropped and is redirected to the `stable` revision.
+To opt-in services for gradual rollouts, a label must be assigned to them. If a
+new revision with no traffic is found in these services, the Release Manager
```

Comment:
If a service has a newly deployed revision with 0% traffic, the [...] (by default, 5%).